www_src/pages/map/map.less

```diff
@@ -100,11 +62,11 @@
   }
 
   .add-page {
-    width: calc(100% - 2px);
-    height: calc(100% - 2px);
-    border: 0;
-    background: #99cccc url('img/add.png') center center no-repeat;
-    background-size: contain;
+    width: 90%;
+    height: 90%;
+    background: #DCE0E6 url('img/plus.svg') center center / 45% no-repeat;
+    border: 1px solid rgba(35,31,32,0.02);
```

Comment:
ricardo decided to remove this, he forgot to update a couple of the comps though